packages/lfrqa/pyproject.toml

```
@@ -0,0 +1,30 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools>=64", "setuptools_scm>=8"]
+
+[project]
+authors = [
+    {email = "hello@futurehouse.org", name = "FutureHouse technical staff"},
+]
+dependencies = [
+    "aviary.litqa",
+    "fh-llm-client>=0.0.11",
+    "fhaviary",
+    "paper-qa",
```

Specify paper-qa to get the last release with the changes you made in the manifest
```suggestion
    "paper-qa>=5.12.0", # For new manifest
```